test/e2e/common/05_tektonhubdeployment_test.go

```diff
@@ -110,6 +110,8 @@ func (s *TektonHubTestSuite) SetupTest() {
 	s.logger.Debug("removing the tekton hub cr if any")
 	s.undeploy("")
 	s.undeployExternalDatabase()
+	// wait for target namespace to be deleted
+	time.Sleep(120 * time.Second)
```

Comment:
there is a helper function for this - `resources.WaitForNamespaceDeletion()` or something, please look it up

Reply:
okay. thanks @concaf